packages/core/src/layout/SignInPage/SignInPage.tsx

```diff
@@ -150,7 +162,7 @@ export const SingleSignInPage = ({
         </Grid>
       </Content>
     </Page>
-  );
+  ) : null;
```

Comment:
Would it be better to render a `<Progress />` from core, which has a timeout that makes it only appear after a certain amount of time has passed? That may be a good fit here if the actual sign-in (on a slow network or so) actually takes a little bit of time.

Reply:
Yeah - better than empty page.